src/main/java/io/vertx/core/metrics/impl/DummyVertxMetrics.java

```diff
@@ -109,7 +111,9 @@ public boolean isMetricsEnabled() {
     return false;
   }
 
-  protected class DummyEventBusMetrics implements EventBusMetrics<Void> {
+  public static class DummyEventBusMetrics implements EventBusMetrics<Void> {
```

Comment:
Nice move on making all those subclasses public here. Dummy impl is useful if you want to partially implement the SPI without having to recompile your code when a new method is added.